notebooks/scripts/plot_effect_estimation.py

```
@@ -42,30 +42,60 @@
+fig, ax = plot_effect_estimation(
     df_patients,
-    x_var="N",
+    x_var="patient_number",
     x_label="Number of patients",
-    filename="effect_estimation_patients_02.png",
+    filename="effect_estimation_patients_amlodipine_a_2p0.png",
     xticks=[200, 1000, 2000, 3000, 4000, 5000, 6000, 7000, 8000],
-    yticks=[0, 0.1, 0.2, 0.3, 0.4],
+    yticks=[-0.3, -0.2, -0.1, 0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8],
 )
+fig
```

Ruff

Found useless expression. Either assign it to a variable or remove it.

(B018)